### ISSUE TITLE
Support configuring mail body

### ISSUE THREAD

**Issue by williamboman** [SaraVieira/react-social-sharing#27]
Right now the body of mail share buttons is [hard-coded to just contain the link](https://github.com/SaraVieira/react-social-sharing/blob/bcf87901d820eb2e01e2a3cf297179872249bc1e/src/consts.js#L38-L41). I'd like to be able to set a custom body, e.g. via a new `body` prop.

One concern to be considered if this were to be added: the name of the existing `message` prop, which would still control the subject, would become quite ambiguous. Does it control the subject or the body? I see two solutions:

1. keep the existing prop name and a new `body` prop
1. rename the existing `message` prop to `subject`, and add a new `body` prop (breaking change)

**Comment by SaraVieira** [comment]
Hey !

Added in https://github.com/SaraVieira/react-social-sharing/releases/tag/v3.0.0


We now have a subject prop that is the new message.
Message was deleted and now there is body :)

**Comment by williamboman** [comment]
Awesome, that's pretty much exactly as I had imagined :)!

What do you think about always appending the link to the body, to ensure that the link that is to be shared is in the email body? I think the way the component API looks like now (`<Mail subject="Subject" body="Body" link="https://example.com" />`) leads people to believe the link will somehow appear in the body (I would believe so at least). Otherwise we delegate the responsibility of sharing the actual link (which is the main function of this component) to the consumer.

I'm thinking something like this (I'm not sure how well email software interprets `\n` though)

```diff
diff --git a/src/consts.js b/src/consts.js
index 89c158f..0ccb7e3 100644
--- a/src/consts.js
+++ b/src/consts.js
@@ -38,7 +38,7 @@ export default {
   mail: (link, subject, body) =>
     `mailto:?subject=${encodeURIComponent(
       subject || ''
-    )}&body=${encodeURIComponent(body || link || '')}`,
+    )}&body=${encodeURIComponent((body && `${body}\n\n${link}`) || link || '')}`,
   pinterest: (link, message) =>
     `https://pinterest.com/pin/create/button/?url=${encodeURIComponent(
       link || ''

```

**Comment by SaraVieira** [comment]
Makes perfect sense:

I added it here: https://github.com/SaraVieira/react-social-sharing/blob/d861180071853dff0124b9dc6460fdd8c9fdb790/src/consts.js

Also cleaned up by adding default params 👌

Thank you man!

Released in 3.0.1
